docs/tutorials/boot-tutorial.md

```diff
@@ -75,7 +91,7 @@ See the commands below:
 ```sh
 git clone https://gem5.googlesource.com/public/gem5
 cd gem5
-git checkout d40f0bc579fb8b10da7181
+git checkout 90a6e809629c978c2df765eb
```

Comment:
Is it possible to use a tag here (e.g., v20.1.0.2)?

Reply:
updated.